src/main/java/com/esri/samples/na/closest_facility_static/ClosestFacilityStaticSample.java

```diff
@@ -183,7 +185,7 @@ public void start(Stage stage) throws Exception {
 
               // add the found incidents to the list
               for (Feature incidentFeature : incidentsResult) {
-                incidentsList.add(new Incident(incidentFeature.getGeometry().getExtent().getCenter()));
+                incidents.add(new Incident((Point) incidentFeature.getGeometry()));
```

Comment:
don't forget a comment here and above that indicates why we know we can cast to Point in this sample (because our feature layers only have point features).